dces-check-attachment-exists/script.sh

```diff
@@ -90,7 +90,7 @@ while : ; do
             echo "File exists in S3." >&2
         else
             file_exists="false"
-            echo "File does not exist in S3." >&2
```

Comment:
Only comment I have on the code is silly: line 52 should probably quote $DCES_ADMIN_USER in case it contains any funny characters.

If this were going to be a long-lived tool used by people outside the team, it might be better re-coded in an easier-to-maintain and secure language. However, as a short-lived ad-hoc script to do a team-only job a handful of times -- it's fine for that.